lms/djangoapps/commerce/utils.py

```diff
@@ -12,6 +12,41 @@ class EcommerceService(object):
     def __init__(self):
         self.config = CommerceConfiguration.current()
 
+    @property
+    def ecommerce_url_root(self):
+        """ Retrieve Ecommerce service public url root. """
+        return configuration_helpers.get_value('ECOMMERCE_PUBLIC_URL_ROOT', settings.ECOMMERCE_PUBLIC_URL_ROOT)
+
+    def get_absolute_ecommerce_url(self, ecommerce_page_url):
+        """ Return the absolute URL to the ecommerce page.
+
+        Args:
+            ecommerce_page_url (str): Relative path to the ecommerce page.
+
+        Returns:
+            Absolute path to the ecommerce page.
+        """
+        return urljoin(self.ecommerce_url_root, ecommerce_page_url)
+
+    def get_receipt_page_url(self, order_number):
+        """
+        Gets the URL for the Order Receipt page hosted by the ecommerce service.
+
+        Args:
+            order_number (str): Order number.
+
+        Returns:
+            Receipt page for the specified Order.
+        """
+        if self.config.use_ecommerce_receipt_page:
```

Comment:
I am re-thinking this (again).  This should just check the site config, if it has a page defined, use it otherwise use the default LMS one.  This way, no need for a new attribute in the commerce model.